config/locales/en.yml

```diff
@@ -212,10 +212,13 @@ en:
     lawfulness_certificate:
       existing: LDCE
       proposed: LDCP
+    prior_approval:
+      proposed: ''
```

Comment:
If only lawfulness certificates have an existing/proposed distinction then we might want to refactor this in future so we don't have to define meaningless translation keys for everything. But that's probably out of scope for now.